dandischema/tests/test_metadata.py

```diff
@@ -44,13 +44,40 @@ def test_dandiset(schema_dir):
 def test_pydantic_validation(schema_dir):
     with (METADATA_DIR / "meta_000004.json").open() as fp:
         data_as_dict = json.load(fp)
+    data_as_dict["schemaVersion"] = "0.4.4"
```

Comment:
why just not to modify in the file since that schema version is not supported?
```
$> grep schemaVersion ./dandischema/tests/data/metadata/meta_000004.json
  "schemaVersion": "0.4.0",
```
and if there is a test to verify that unsupported causes error, only then do explicit resetting of the version with a comment on purpose .
Otherwise later requires "thinking" on why it is reset here

Reply:
I think Satra has RFed for this, can be resolved